<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Veracross to ADP Sync</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="debugTab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>751</width>
      <height>481</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="syncTab">
     <attribute name="title">
      <string>Sync</string>
     </attribute>
     <widget class="QPushButton" name="getVCDataButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Get Veracross Data</string>
      </property>
     </widget>
     <widget class="QLabel" name="logo_label">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>10</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="vcDataFSStatusLabel">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>20</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="vcFSRecordCount">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0 Faculty Staff Records</string>
      </property>
     </widget>
     <widget class="QPushButton" name="parseVCDataButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Parse Veracross Data</string>
      </property>
     </widget>
     <widget class="QLabel" name="vcParseStatusLabel">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>80</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>getVCDataButton</zorder>
     <zorder>logo_label</zorder>
     <zorder>vcDataFSStatusLabel</zorder>
     <zorder>parseVCDataButton</zorder>
     <zorder>vcParseStatusLabel</zorder>
     <zorder>vcFSRecordCount</zorder>
    </widget>
    <widget class="QWidget" name="resultsTab">
     <attribute name="title">
      <string>Results</string>
     </attribute>
     <widget class="QTextEdit" name="resultsTextEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>721</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_ResultsPrevious">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Previous</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_ResultsNext">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="settingsTab">
     <attribute name="title">
      <string>Settings</string>
     </attribute>
     <widget class="QLineEdit" name="vc_api_user">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>30</y>
        <width>291</width>
        <height>21</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::PasswordEchoOnEdit</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_vcapiusername">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Veracross API Username</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="vc_api_pass">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>70</y>
        <width>291</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::PasswordEchoOnEdit</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_vcpassword">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Veracross API Password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="vc_api_url">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>110</y>
        <width>291</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_vcapiurl">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Veracross API Base URL</string>
      </property>
     </widget>
     <widget class="QPushButton" name="settingsSave">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>410</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Debug</string>
     </attribute>
     <widget class="QLabel" name="lbl_vcLastXRateLimitReading">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>301</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Veracross Last X-Rate-Limit-Remaining Reading</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditXRateLimitReading">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>10</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditXRateLimitResetReading">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>40</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_vcLastXRateResetReading">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>301</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Veracross Last X-Rate-Limit-Reset Reading</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textLog">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>731</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_Log">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Log</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
